src/script/scripting_client.cpp

```diff
@@ -36,6 +36,7 @@ ClientScripting::ClientScripting(Client *client):
 	ScriptApiBase()
 {
 	setGameDef(client);
+	setType(ScriptingType::Client);
```

Comment:
I think I'm missing something but doesn't that mean it also has to be a constructor argument for all class that inherit from `ScriptApiBase`? That is an issue because all the C++ -> Lua API class inherit from it.